src/UI/Component/Panel/Listing/Divider.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/* Copyright (c) 2017 Alex Killing <killing@leifos.de> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Component\Panel\Listing;
+
+/**
+ * Listing divider
+ * @package ILIAS\UI\Component\Panel\Listing
+ */
+interface Divider extends \ILIAS\UI\Component\Component {
```

Comment:
> We go with option 3

Works for me. Is this a) also the suggested solution for dropdowns, or do b) we keep the dividers there? If a) do we need a special group for this (since there we do not have a title or such)? If b) I think then we can simplify the dividers further since they do not need a title or such.

> No separate Heading component. We will keep trying to find a way to deal with the heading hierarchy.

This seems to be a nutcase. 

> Groups will be in the Item namespace: Item > Group

With the issue that this could be read as "Group Item" which to me sound like an item containing information about a group. However, throwing it on the top level or any place else seems also to be odd. Maybe changing the name to "Grouping" (the noun not the verb, "Die Gruppenbildung") would help with this. Then it reads "Grouping Item" (an item that groups stuff). However since this is not really an item the problem somehow remains. Nevertheless, I can live with it.

> Listings accept an array of Item Groups only in the constructor for now (if we really have the need to add single items here, we may do this in the future, but I think this is not the case).

Definitely!

Thank you both for your endurance in finding a good solution.